snmp/datadog_checks/snmp/data/conf.yaml.example

```
@@ -303,3 +303,20 @@ instances:
         name: tcpPassiveOpens
         metric_tags:
           - TCP
+
+    ## @param metric_tags - list of elements - optional
+    ## Specify tags that you want applied to all metrics. It can be a symbol or a OID.
+    ##
+    ## - MIB: SNMPv2-MIB
+    ##   symbol: sysName
+    ##   name: snmp_host
```

Should we use `tag` here, instead of `name`? My main motivation is that this format is close to, but different from, the way we specify metrics in profiles, so using a `tag` key might be clearer that the value will be the name of the tag, instead of some alternative name of the OID.

```suggestion
    ##   tag: snmp_host
```

---

Sounds good.